src/Logger.js

```diff
@@ -62,81 +63,109 @@ export default class Logger {
       return acc;
     }, {});
 
-    this.winston = new winston.Logger({
+    this.winston = winston.createLogger({
       transports: [
         new winston.transports.Console({
-          stderrLevels: this.stderrLevels,
-          handleExceptions: false,
-          colorize: this.colorize,
-          formatter: this.isProduction
-            ? (...args) => this.formatJSONMessage(...args)
-            : undefined,
-          level: this.level
+          stderrLevels: this.stderrLevels
         })
       ],
+      format: this.format(),
+      level: this.level,
       levels: this.winstonLevels,
-      colors: this.colors
+      handleExceptions: false,
+      exitOnError: this.exitOnError
     });
-
-    // winston doesn't allow to set the error level for runtime
-    // exceptions, therefore this should be handled outside of winston.
-    // https://github.com/winstonjs/winston/blob/bd06c4091e9e248e6fed6278e0699e5cfdc7de24/lib/winston/exception-handler.js#L94-L97
-    process.on('uncaughtException', this.onUnhandledException);
-
-    // Unhandled promise rejections should be treated as errors.
-    process.on('unhandledRejection', this.onUnhandledException);
   }
 
   createLogMethods() {
-    Object.keys(this.levels).forEach(level => {
-      this[level.toLowerCase()] = message => {
+    Object.keys(this.levels).forEach((level) => {
+      this[level.toLowerCase()] = (message) => {
         let meta;
-
         // Log functions can be passed an object to provide
         // additional meta data like a logger `name`.
         if (typeof message === 'object') {
           meta = {...message};
           delete meta.message;
           message = message.message;
         }
-
-        this.winston.log(level, message, meta);
+        // Pass the meta object as actual `meta` object
+        this.winston.log(level, message, {meta});
```

Comment:
This would be a chance to get rid of `meta`, as [we want to move away from that anyway](https://bitbucket.org/setlist/setlist-widget-festival/pull-requests/23/sfm-2934-forward-request-id#comment-431441666).

But only if you think it's a good time to adapt this, I don't want to scope creep here too much.

Reply:
I'll leave this up to you if you want to address this!